### PR TITLE
add fetch raw data

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: fetch_raw_data.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+from core_data_modules.util import IOUtils, TimeUtils, SHAUtils
+from id_infrastructure.firestore_uuid_table import FirestoreUuidTable
+from rapid_pro_tools.rapid_pro_client import RapidProClient
+from storage.google_cloud import google_cloud_utils
+from temba_client.v2 import Contact, Run
+
+from src.lib import PipelineConfiguration
+from src.lib.pipeline_configuration import RapidProSource
+
+Logger.set_project_name("Wusc-Keep-II")
```
Comment (as2388) [inline fetch_raw_data.py]:
```suggestion
Logger.set_project_name("WUSC-KEEP-II")
```
(This is the project name on GitHub)

**Review by as2388 — APPROVED** [review]
Approved if this is copied from WorldBank-PLR with only the image and project names changed.

When making large copy-paste changes like this it would be helpful if you could add a small note to say if anything beyond the project names changed.

File: fetch_raw_data.py (hunk 1 above)
Comment (IsaackMwenda) [reply fetch_raw_data.py]: The main difference being this does not support Shaqadoon and google cloud bucket as raw data sources.

**Comment by IsaackMwenda** [comment]
> Approved if this is copied from WorldBank-PLR with only the image and project names changed.
> 
> When making large copy-paste changes like this it would be helpful if you could add a small note to say if anything beyond the project names changed.

The main difference being this does not support Shaqadoon and google cloud bucket as raw data sources.

**Pull request merged**